tck/src/main/java/org/eclipse/microprofile/config/tck/configsources/SampleYamlConfigSource.java

```diff
@@ -46,6 +47,16 @@ public String getValue(String key) {
         return config.get(key);
     }
 
+    @Override
+    public ConfigValue getConfigValue(final String propertyName) {
+        return ConfigValue.builder()
```

Comment:
That just doesn't look very well performing to be honest!

Reply:
This is just a TCK and it was an easy fix to compile the project.